analytics/CMakeLists.txt

```diff
@@ -104,6 +104,9 @@ set_property(TARGET firebase_analytics PROPERTY FOLDER "Firebase Cpp")
 # Set up the dependency on Firebase App.
 target_link_libraries(firebase_analytics
     PUBLIC firebase_app)
+if(WIN32)
+  target_link_libraries(firebase_analytics PUBLIC Crypt32.lib)
```

Comment:
Does this need to be PUBLIC?  Looking over other cases, we usually try to make these private, though there might be a reason this one needs to be public

Reply:
Let's try it and see what happens!